addons/web/static/src/js/widgets/colorpicker.js

```diff
@@ -1,22 +1,22 @@
-odoo.define('web.ColorpickerDialog', function (require) {
+odoo.define('web.Colorpicker', function (require) {
```

Comment:
Don't force me to repeat that again please ;) You move/rename a file? You do that in a dedicated commit.